packages/bpk-docs/src/pages/NativeSpinnerPage/NativeSpinnerPage.js

```diff
@@ -0,0 +1,102 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import BpkLink from 'bpk-component-link';
+import readme from 'react-native-bpk-component-spinner/readme.md';
+
+import iosScreenshotDefault from 'react-native-bpk-component-spinner/screenshots/ios/default.png';
+import iosScreenshotSmall from 'react-native-bpk-component-spinner/screenshots/ios/small.png';
+import androidScreenshotDefault from 'react-native-bpk-component-spinner/screenshots/android/default.png';
+import androidScreenshotSmall from 'react-native-bpk-component-spinner/screenshots/android/small.png';
+
+import Paragraph from './../../components/Paragraph';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+import { THEMING } from './../../constants/routes';
+
+const reactNativeUrl = 'https://facebook.github.io/react-native/docs/activityindicator.html';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: 'By default, spinners come in a large size and are available in three types: primary, dark and light.', // eslint-disable-line max-len
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Spinner Component',
+        subText: '(iPhone 8 Simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotDefault}`,
+        altText: 'Android Default Spinner Component',
+        subText: '(Google Pixel Emulator)',
+      },
+    ],
+  },
+  {
+    id: 'small',
+    title: 'Small',
+    blurb: 'Spinners can also appear in a smaller size in the same types as mentioned above.',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotSmall}`,
+        altText: 'iOS Small Spinner Component',
+        subText: '(iPhone 8 Simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotSmall}`,
+        altText: 'Android Small Spinner Component',
+        subText: '(Google Pixel Emulator)',
+      },
+    ],
+  },
+];
+
+const NativeSpinnerPage = () => (<DocsPageBuilder
+  title="Spinners"
+  blurb={[
+    <Paragraph>
+      Spinners are used to to indicate that a part of the product is loading or performing a task.
+      These are available in two sizes, small and large. It&#39;s a wrapper around the React
```

Comment:
do we need this `It&#39;s` or it can be a simple `'`?

Reply:
Better safe than sorry I guess!

Reply:
it was a genuine question, I've never done it before and I wanted to know if I have to use from now on 😄

Reply:
Oh haha, well the reason I have it is because I copied it from an old docs page. So I figured I'd continue to use it for consistency.

Reply:
![image](https://user-images.githubusercontent.com/3579758/32320477-cc60f862-bfb5-11e7-903a-5b19846accde.png)